extensions/panache/hibernate-orm-panache/deployment/src/test/java/io/quarkus/hibernate/orm/panache/deployment/test/NoConfigTest.java

```diff
@@ -1,20 +1,32 @@
 package io.quarkus.hibernate.orm.panache.deployment.test;
 
-import org.jboss.shrinkwrap.api.ShrinkWrap;
-import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import jakarta.persistence.Entity;
+import jakarta.persistence.NamedQuery;
+
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
+import io.quarkus.hibernate.orm.panache.PanacheEntity;
 import io.quarkus.test.QuarkusUnitTest;
 
 public class NoConfigTest {
 
     @RegisterExtension
-    static final QuarkusUnitTest config = new QuarkusUnitTest().setArchiveProducer(
-            () -> ShrinkWrap.create(JavaArchive.class));
+    static QuarkusUnitTest runner = new QuarkusUnitTest()
+            .withApplicationRoot((jar) -> jar
+                    .addClasses(MyEntity.class));
 
     @Test
-    public void testNoConfig() {
-        // we should be able to start the application, even with no configuration at all
```

Comment:
Are you sure you want to repurpose this test, without even changing its name?

Reply:
OK, should be fixed. Thanks for spotting this, that was embarrassing :(